### PR TITLE
Add Juho's recipe draft

### PR DESCRIPTION
**Pull request by juholehtonen**
The recipe is cooking and follows.

The reasons for changes:
- reasons 1
- reasons 2

### CODE REVIEW

**Comment by Sabryr** [comment]
Referring to issue  #2

**Comment by Sabryr** [comment]
closes #2

File: juhos-recipe.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Here I will write my favorite recipe...!
+
```
Comment (Sabryr) [inline juhos-recipe.txt:2]: Can you please send me the actual recipe

**Review by Sabryr — APPROVED** [review]

File: juhos-recipe.txt (hunk 1 above)
Comment (juholehtonen) [reply juhos-recipe.txt:2]: Sure! Working on it!

**Pull request merged**